src/types/Firestore.ts

```diff
@@ -80,12 +80,17 @@ export interface FirestoreOrdered {
   eventPurchase: Array<WithId<Purchase>>;
   events?: Array<WithId<VenueEvent>>;
   experiences: Array<WithId<Experience>>;
+  parentVenueEvents: Array<WithId<VenueEvent>>;
   partygoers: Array<WithId<User>>;
   playaVenues?: Array<WithId<AnyVenue>>;
   privatechats: Array<WithId<PrivateChatMessage>>;
   reactions: Array<WithId<Reaction>>;
+  siblingVenues: Array<WithId<AnyVenue>>;
+  siblingVenueEvents: Array<WithId<VenueEvent>>;
   statsOnlineUsers?: Array<WithId<User>>;
   statsOpenVenues?: Array<WithId<AnyVenue>>;
+  subvenues: Array<WithId<AnyVenue>>;
+  subvenueEvents: Array<WithId<VenueEvent>>;
```

Comment:
See related comments RE: unique names for the `.map()`'d queries

Reply:
Also, all of these types are wrong (see https://github.com/sparkletown/sparkle/pull/744), so maybe add your new ones with `?` to at least avoid potential issues from them.

Reply:
```suggestion
  parentVenueEvents?: Array<WithId<VenueEvent>>;
  partygoers: Array<WithId<User>>;
  playaVenues?: WithId<AnyVenue>;
  privatechats: Array<WithId<PrivateChatMessage>>;
  reactions: Array<WithId<Reaction>>;
  siblingVenues?: WithId<AnyVenue>[];
  siblingVenueEvents?: WithId<VenueEvent>[];
  statsOnlineUsers?: Array<WithId<User>>;
  statsOpenVenues?: Array<WithId<AnyVenue>>;
  subvenues: WithId<AnyVenue>[];
  subvenueEvents: WithId<VenueEvent>[];
```